tests/tests.go

```diff
@@ -0,0 +1,36 @@
+package tests
```

Comment:
Not sure on the package structure here. This is helper code for the other tests?

Reply:
Exactly, I wasn't sure where to put it, suggestions more than welcome

Reply:
Generally, we try and extract packages to a separate repo for reuse. Since this is generic k8s `operatorkit` is an option but we do want to keep it lean. I'm OK with it here for now.

Reply:
It'd be ok to move that to the operatorkit but it's a hack. We just utilise the fact that K8s client doesn't do a request upon creation with non-incluster config.